test/validation/api/time/time.c

```diff
@@ -28,6 +34,96 @@ typedef uint64_t time_res_cb(void);
 typedef odp_time_t time_from_ns_cb(uint64_t ns);
 typedef uint64_t time_nsec_cb(void);
 
+typedef struct {
+	uint32_t num_threads;
+	odp_barrier_t test_barrier;
+	odp_time_t time[MAX_WORKERS + 1][TIME_SAMPLES];
+} global_shared_mem_t;
+
+static global_shared_mem_t *global_mem;
+static odp_instance_t *instance;
+
+static int time_global_init(odp_instance_t *inst)
+{
+	odp_shm_t global_shm;
+	odp_init_t init_param;
+	odph_helper_options_t helper_options;
+	uint32_t workers_count, max_threads;
+
+	if (odph_options(&helper_options)) {
+		fprintf(stderr, "error: odph_options() failed.\n");
+		return -1;
+	}
+
+	odp_init_param_init(&init_param);
+	init_param.mem_model = helper_options.mem_model;
+
+	if (0 != odp_init_global(inst, &init_param, NULL)) {
+		fprintf(stderr, "error: odp_init_global() failed.\n");
```

Comment:
`ODPH_ERR()` could be used for error messages. I'll do a PR which fixes this for all validation tests.

Reply:
In v6. Also added a commit which changes the existing fprintf() calls in this test to ODPH_ERR(), but this can be dropped if you already have it.

Reply:
The commit is OK here. I'll just rebase my PR on top of this one.